src/util/saveCurrentCursorOffset.js

```diff
@@ -0,0 +1,8 @@
+import debounce from 'lodash.debounce';
+
+export default function saveCurrentCursorOffset() {
+	debounce(() => {
+		const offset = window.getSelection().focusOffset;
+		localStorage.setItem('currentCursorPosition', offset);
+	}, 10)();
```

Comment:
This won't actually debounce the function the way it is implemented. It will generate a new debounced function every time `saveCurrentCursorOffset` is called. Instead, you need to debounce `saveCurrentCursorOffset` itself, or the function that calls it and preserve the debounced function reference.

See https://github.com/cybersemics/em/blob/dev/src/components/Search.js#L28-L30 for an example.

Reply:
oh, yeah. I don't know what I was thinking about when writing this. I'll redo! Thanks